src/pages/MyPage/MyPage.js

```diff
@@ -0,0 +1,148 @@
+import styled from 'styled-components';
+import { MdAlarmOn, MdCheckCircleOutline, MdBlock } from 'react-icons/md';
+
+const flexCenter = `
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const S = {
+  AuctionBoxWrapper: styled.div`
+    gap: 18px;
+    width: 930px;
+    background-color: ${props => props.theme.darkGrey};
+    border-radius: 10px;
+    ${flexCenter}
```

Comment:
```suggestion
  AuctionBoxWrapper: styled.div`
      ${flexCenter};
    gap: 18px;
    width: 930px;
    background-color: ${props => props.theme.darkGrey};
    border-radius: 10px;
```
mixin은 scss와 마찬가지로 가장 위에 위치하는 것이 좋습니다.